missas/core/tests/test_migration_0030_migrate_schedule_location_to_location_table.py

```
@@ -0,0 +1,85 @@
+    OldSchedule = old_state.apps.get_model(app_name, "Schedule")
+    location_name = "Main Church"
+    schedule1 = baker.make(OldSchedule, location=location_name, parish__name="Parish 1")
+    schedule2 = baker.make(OldSchedule, location=location_name, parish__city=schedule1.parish.city, parish__name="Parish 2")
+
+    new_state = migrator.apply_tested_migration(migration_full_name)
+
+    NewLocation = new_state.apps.get_model(app_name, "Location")
+    locations = NewLocation.objects.all()
+
+    assert NewLocation.objects.count() == 2
+    assert {l.slug for l in locations} == {"parish-1-main-church", "parish-2-main-church"}
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)